_layouts/base.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <title>{{ site.title }}</title>
+        <link rel="shortcut icon" type="image/png" href="/images/favicon.ico">
+        <link rel="stylesheet" href="/css/bootstrap.min.css">
+        <link rel="stylesheet" href="/css/mitoc.css">
+        <script defer src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.2/jquery.min.js" integrity="sha384-mXQoED/lFIuocc//nss8aJOIrz7X7XruhR6bO+sGceiSyMELoVdZkN7F0oYwcFH+" crossorigin="anonymous"></script>
```

Comment:
This is a really old version of jquery - I'm assuming we don't care though. Does it make sense to locally host it?

Reply:
I didn't change that from the original HTML pages. I'd not upgrade now to avoid breaking something, but if we want to add more interactive features to the website, using more modern front-end tools would probably be relevant.
Hosting using the Google CDN theoretically help with cache and loading time, though with such an outdated library not sure it does much...